src/main/java/com/mineboundteam/minebound/event/GlowingEvent.java

```diff
@@ -0,0 +1,58 @@
+package com.mineboundteam.minebound.event;
+
+import org.lwjgl.glfw.GLFW;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.registry.KeyRegistry;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.OutlineBufferSource;
+import net.minecraft.client.renderer.entity.EntityRenderDispatcher;
+import net.minecraft.util.Mth;
+import net.minecraft.world.entity.LivingEnity;
```

Comment:
```suggestion
import net.minecraft.world.entity.LivingEntity;
```
This somehow got typod to LivingEnity.